modules/openapi-generator/src/main/resources/dart2/object_doc.mustache

```diff
@@ -1,14 +1,14 @@
-{{#models}}{{#model}}# {{pubName}}.model.{{classname}}
+{{#models}}{{#model}}# {{{pubName}}}.model.{{classname}}
 
 ## Load the model package
 ```dart
-import 'package:{{pubName}}/api.dart';
+import 'package:{{{pubName}}}/api.dart';
```

Comment:
This import using `'` again - which I think is correct, but pointing out lack of consistency.